docs/user-guide/reference/autoscaling.md

```diff
@@ -60,8 +61,11 @@ spec:
   support autoscaling, typically increasing the value leads to lower processing rate, thus less replicas.
 - `targetBufferAvailability` - Targeted buffer availability in percentage, defaults to `50`. It is only effective for `UDF`
   and `Sink` vertices, it determines how aggressive you want to do for autoscaling, increasing the value will bring more replicas.
-- `replicasPerScale` - Maximum number of replicas change happens in one scale up or down operation, defaults to `2`. For
+- `replicasPerScaleUp` - Maximum number of replicas change happens in one scale up operation, defaults to `2`. For
```

Comment:
`Maximum number of replica change that  happens in one scale up operation, defaults to `2``